tracing-subscriber/CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# 0.3.18 (October 12, 2023)
+
+### Fixed
+- fix new warnings in Rust 1.72.0 (#2700)
+- fix clippy lints (#2673)
+- use fully qualified names in macros for items exported from std prelude (#2621)
```

Comment:
i don't think this change is actually part of the subscriber release --- i think it just touched tests?

Reply:
ah, true—i didn't look closely enough. only saw that it was impacted.